components/tidb_query_datatype/src/codec/data_type/chunked_vec_bool.rs

```diff
@@ -0,0 +1,228 @@
+// Copyright 2020 TiKV Project Authors. Licensed under Apache-2.0.
+
+#[derive(Debug, PartialEq, Clone)]
+pub struct ChunkedVecBool {
+    data: Vec<u8>,
+    length: usize,
+}
+
+impl ChunkedVecBool {
```

Comment:
Maybe call it `BitVec`? (And do we have some existing BitVec to use?)

Reply:
Alright, I'll change it to `BitVec`. 

I prefer to write our own version of `BitVec` because later we'll introduce merge nulls before loop feature for non-nullable vectorized functions. This can be done by simply checking if any item in one row is `None`. A faster implementation, that we directly apply bitwise "or" on u8 type, requires access to internal structures of BitVec. Existing BitVec implementation may not provide access to internal vector.

Reply:
Changed to `BitVec`